dolphin-dnn/src/test/java/edu/snu/dolphin/dnn/NeuralNetworkTest.java

```diff
@@ -58,17 +58,17 @@
 
   private final Matrix input = matrixFactory.create(new float[]{77, 57, 30, 26, 75, 74, 87, 75});
   private final Matrix expectedOutput =
-      matrixFactory.create(new float[]{5.61329743164e-01f, 5.97883503916e-01f, 4.79822915984e-01f});
+      matrixFactory.create(new float[]{5.96001906923e-01f, 5.54388444438e-01f, 4.88766051614e-01f});
```

Comment:
The shapes of fully connected layers' weight matrices are changed from `numInput x numOutput` to `numOutput x numInput`. Thus, the positions of generated random numbers in the matrix are changed, which generates different outputs.
FYI, if using weight matrices generated by `matrixFactory.randn(numInput, numOutput, randomSeed).transpose()` in `FullyConnectedLayerParameterInitializer`, we can get the same results. But, we should avoid using it because `transpose()` is a costly function.